.github/workflows/test-js-packages.yml

```diff
@@ -289,35 +299,28 @@ jobs:
           name: noir_wasm
           path: ./compiler/wasm
 
+      - name: Install Yarn dependencies
+        uses: ./.github/actions/setup
+
       - name: Download nargo binary
         uses: actions/download-artifact@v3
         with:
           name: nargo
           path: ./nargo
 
-      - name: Compile fixtures with Nargo CLI
-        working-directory: ./compiler/wasm/fixtures
+      - name: Set nargo on PATH
         run: |
-          nargo_binary=${{ github.workspace }}/nargo/nargo
+          nargo_binary="${{ github.workspace }}/nargo/nargo"
           chmod +x $nargo_binary
-          for dir in $(ls -d */); do
-            pushd $dir/noir-script
-            $nargo_binary compile
-            popd
-          done
+          echo "$(dirname $nargo_binary)" >> $GITHUB_PATH
+          export PATH="$PATH:$(dirname $nargo_binary)"
 
-      - name: Install Yarn dependencies
-        uses: ./.github/actions/setup
-
-      - name: Install Playwright
-        uses: ./.github/actions/install-playwright
+      - name: Build fixtures
+        run: yarn workspace @noir-lang/noir_wasm test:build_fixtures
 
       - name: Run node tests
         run: yarn workspace @noir-lang/noir_wasm test:node
 
-      - name: Run browser tests
-        run: yarn workspace @noir-lang/noir_wasm test:browser
```

Comment:
After inlining the wasm artifact and some other compilation changes, I was able to move the complete (meaning not only wasm but also dependency resolution) tests to the `noir-wasm` package. 

It's kind of a hack because I can't get TS to pick up the type declarations from the compiled bundle, but the overall solution is much cleaner. Now the testing structure is:

Tested in "isolation" in `noir-wasm`: 
* Dependency resolvers
* wasm compilation in node and the browser (output of wasm-pack)

Tested after bundling in `noir-wasm`
* Project compilation with dependency resolution in node and the browser

Tested in the `integration-tests`
* Project compilation + verification + recursion